frontend/src/app/search/_search.scss

```diff
@@ -0,0 +1,36 @@
+@use "uswds-core" as *;
+
+// Define border color as a variable
+$border-color: #3b48d8;
+
+// Mixin for the border for reuse
+@mixin border-style {
+  border: 1px solid $border-color;
+}
+
+.grid-container {
+  @include border-style;
+
+  .grid-row {
+    @include border-style;
+
+    .grid-col {
+      @include border-style;
+    }
+  }
+}
```

Comment:
add border around everything so it's easier to see the containers while building